### DISCUSSION TITLE
scrum prep Milestone 2 - 5

### DISCUSSION THREAD

**Discussion by Yahid1** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#144] [General]
Write your scrum prep here.

**Comment by MrL9000** [comment]
I finished my work early, on last week, all that is left is a task that is to be moved to the next milestone, next week I'll plan out next milestone's tasks, once the objectives have been decided on.

**Comment by AleDiazz** [comment]
finished my tasks for this milestone, and did my 2 out of 4 lecture topic task for the project. Next week for milestone 3 i will get new tasks

**Comment by yamayrie** [comment]
worked on a lecture topic task and also fixed a documentation section based on feedback

**Comment by gabrielpadilla7** [comment]
Worked on a lecture topic task, closed issues, monitored the changes in the documentation (following guidelines).

**Comment by JaseyAcevedo** [comment]
Finished tasks and ltt

**Comment by Yahid1** [comment]
For this week I assigned some task, closed some tasks. Clarified all tasks if confusion around tasks. Coordinated with the RUMBus team to incorporate some of the changes into the official RUMBus git repository.

**Comment by Brandell13** [comment]
I was working with my lecture topic task and analyzing future tasks that could be worked on for the next milestone

**Comment by diegorobles2** [comment]
This week I finished all my tasks

**Comment by IvanierB** [comment]
This week I completed a Lecture Topic Task, worked on documentation tasks, fixing sections based on feedback. I also worked on a coding task that its almost done, just waiting on some bug fixes in the backend so I can test my code.

**Comment by sebasagir** [comment]
started working on llt

**Comment by Alej24** [comment]
During this week I closed the two milestone tasks I hadn't finished.

**Comment by rafa-quiles** [comment]
Worked on lecture task

**Comment by juan-ortiz92** [comment]
This week I have been working on finishing my issues and working on my third lecture topic task.

**Comment by marcosarraiza** [comment]
This week I worked alongisde William to close a lecture topic task where we refractored the vehicle page.

**Comment by AlexisTraversoPR** [comment]
This week I worked mostly on learning about bounded contexts for lecture topic tasks and adding to documentation

**Comment by JoelTorres20** [comment]
This week I finally finished my debugging assignment (which took me 20+ hours to do) and I also created and finished my second lecture topic task.

**Comment by dAninAd1248** [comment]
I connected all the views under the vehicles page.

**Comment by JConcepcion01** [comment]
This week I worked on finishing my tasks for milestone 2

**Comment by Janiel777** [comment]
En esta semana estuve investigando y preparando unos issues de LTT.

**Comment by dylanDeJesus21** [comment]
Finished modifying the front end for the selected ownership creation. Worked on documentation expanding on what libraries are used throughout the app. Created a lecture topic task on explicit constraints
